Documentation/building/debugging-instructions.md

```diff
@@ -127,18 +127,57 @@ However the common commands have been aliased so that you don't need the SOS pre
     pe              -> sos PrintException
     soshelp         -> sos Help
 
+### Debugging core dumps with lldb
+
+It is also possible to debug .NET Core crash dumps using lldb and SOS. In order to do this, you need all of the following:
+
+- The crash dump file. We have a service called "Dumpling" which collects, uploads, and archives crash dump files during all of our CI jobs and official builds.
+- On Linux, there is an utility called `createdump` (see [doc](https://github.com/dotnet/coreclr/blob/master/Documentation/botr/xplat-minidump-generation.md "doc")) that can be setup to generate core dumps when a managed app throws an unhandled exception or faults.
+- Matching coreclr/corefx runtime bits from the crash. To get these, you should either:
+  - Download the matching Jenkins archive onto your repro machine.
+  - Check out the coreclr and corefx repositories at the appropriate commit and re-build the necessary portions.
+  - You can also download the matching "symbols" nuget package from myget.org. There is a "Download Symbols" button in the myget UI for this purpose.
+- lldb version 3.9. Versions 3.6+ of lldb work with regular debugging, but not core debugging. Make sure the version of lldb you have installed is >= 3.8.
+- libsosplugin.so built against a matching version of lldb. Unfortunately, the one that is included in the CoreCLR nuget package is built against version 3.6. You will need to build coreclr from source to get the correct version. Luckily, this will help you get the next file:
```

Comment:
These sentences are not complete. Maybe we should just remove them and  be explicit about the assumption that whoever is using these instructions has already built the CoreCLR binaries himself?